featurebyte/routes/feature_list_namespace/controller.py

```diff
@@ -85,6 +86,7 @@ async def get(
                 "entity_ids": default_feature_list_doc["entity_ids"],
                 "table_ids": default_feature_list_doc["table_ids"],
                 "readiness_distribution": default_feature_list_doc["readiness_distribution"],
+                "dtype_distribution": default_feature_list_doc["dtype_distribution"],
```

Comment:
We can assume that all feature list documents will have the `dtype_distribution` field populated already right?

Reply:
Yes, after migration all feature list documents should have this field. If the mongo has not been gone through migration, it will get migrated automatically during startup.